pkg/fs/realfs.go

```diff
@@ -114,7 +116,13 @@ func (realFS) IsDir(name string) bool {
 }
 
 // ReadFile delegates to ioutil.ReadFile.
-func (realFS) ReadFile(name string) ([]byte, error) { return ioutil.ReadFile(name) }
+func (realFS) ReadFile(name string) ([]byte, error) {
+	file, err := ioutil.ReadFile(name)
+	if filepath.Ext(name) == "json" {
+		file, err = yaml.JSONToYAML(file)
```

Comment:
Since JSON is a subset of YAML we don't actually need to do any conversion for the file contents to be unmarshaled.

Reply:
Thanks for the information. I assumed that reading JSON was not supported since YAML is specifically mentioned in the docs while JSON isn't.
I've confirmed now that the original code reads JSON as well so I will remove this code.